vishwamai/deepthinking.py

```
@@ -9,6 +9,7 @@
 from collections import defaultdict
 from .model import VishwamaiModel
 from .conceptual_tokenizer import ConceptualTokenizer
+from .architecture import init_model  # Import init_model function
```

Ruff (0.8.2)

`.architecture.init_model` imported but unused

Remove unused import: `.architecture.init_model`

(F401)